ports/mimxrt10xx/common-hal/busio/UART.c

```diff
@@ -202,16 +209,19 @@ void common_hal_busio_uart_construct(busio_uart_obj_t *self,
             }
         }
         if (self->cts == NULL) {
-            mp_raise_ValueError(translate("Selected CTS pin not valid"));
+            mp_raise_ValueError_varg(translate("Invalid %q pin"), MP_QSTR_CTS);
         }
     }
 
     if (self->rx) {
         self->uart = mcu_uart_banks[self->rx->bank_idx - 1];
     } else {
+        assert(self->rx);
```

Comment:
was the assert(self->rx) a known failure, since we just checked self-rx above, so we know it's false
compare that to the self-tx and self-uart usages below